src/Sulu/Bundle/AdminBundle/Resources/js/containers/ViewRenderer/ViewRenderer.js

```diff
@@ -6,13 +6,46 @@ import {sidebarStore} from '../Sidebar';
 import Router from '../../services/Router';
 import type {Route} from '../../services/Router';
 import viewRegistry from './registries/ViewRegistry';
+import type {View} from './types';
 
 type Props = {
     router: Router,
 };
 
 @observer
 export default class ViewRenderer extends React.Component<Props> {
+    componentDidUpdate() {
+        this.checkForSidebar();
+    }
+
+    componentDidMount() {
+        this.checkForSidebar();
+    }
+
+    checkForSidebar() {
```

Comment:
Wouldn't something like `clearSidebarConfig` be a better name? `checkForSidebar` doesn't really say that something is cleared...